### ISSUE TITLE
Licensing of EVE related images and data

### ISSUE THREAD

**Issue by ZeroPointEnergy** [pyfa-org/Pyfa#1505]
Hello

Im the proxy maintainer for the pyfa package for gentoo. Recently the question came up about what the actual licenses are we have to reference with the application.

I know pyfa itself is licensed under the GPL3. But what about the content, the images and eve.db from CCP. Is there a concrete license text for this files we can reference somewhere?

**Comment by blitzmann** [comment]
Hi there @ZeroPointEnergy! It's always awesome to meet the disto maintainers for pyfa, thank you for all the hard work that you do to support pyfa on Gentoo!

As for the license, I'm not 100% sure. I don't believe CCP has every really attached a license to that data, other than the following text:

> EVE Online, the EVE logo, EVE and all associated logos and designs are the intellectual property of CCP hf. All artwork, screenshots, characters, vehicles, storylines, world facts or other recognizable features of the intellectual property relating to these trademarks are likewise the intellectual property of CCP hf. EVE Online and the EVE logo are the registered trademarks of CCP hf. All rights are reserved worldwide. All other trademarks are the property of their respective owners. CCP hf. has granted permission to pyfa to use EVE Online and all associated logos and designs for promotional and information purposes on its website but does not endorse, and is not in any way affiliated with, pyfa. CCP is in no way responsible for the content on or functioning of this program, nor can it be liable for any damage arising from the use of this program.

This is old text that was suggested to include in third party applications by an old support article. I will ping a few of the devs to see if I can get actual licensing information for the data and images provided.

**Comment by ZeroPointEnergy** [comment]
Thanks a lot for inquiring this @blitzmann . And thanks for your amazing work with pyfa! EVE would not be the same without it :smile:

**Comment by blitzmann** [comment]
So, seems the official response is: ¯\_(ツ)_/¯

Slightly longer explanation, for documentation:

![image](https://user-images.githubusercontent.com/3904767/38680737-040eb45e-3e35-11e8-8ff2-e10478137b95.png)

So, there doesn't seem to be any concrete license that you can reference. I'm going to push CCP to include at least some sort of permissive text on the developer resources. Otherwise, simply reference the copyright notice in the README for EVE data attribution

**Comment by blitzmann** [comment]
> I'm not even sure why he gets the impression I asking for compatibility, since that does not even makes sense.

I had the same thought tbh. I know some distros are a bit strict with not including things that are not licensed correctly (or compatible with other FOSS licenses), or at least need to be correctly and explicitly noted.

Sorry I couldn't be of more help though. I could break out the EVE copyright notice from the README to it's own file if that would make things easier with the packaging.

**Comment by ulm** [comment]
&lt;license team&gt;
Looking at the "CCP Copyright Notice" at the bottom of https://github.com/pyfa-org/Pyfa/blob/master/README.md, I see basically that all rights are reserved, followed by this statement: "CCP hf. has granted permission to pyfa to use EVE Online and all associated logos and designs for promotional and information purposes on its website"

This appears to be a permission specific for their website. I don't think that it would allow us (nor anybody else) to distribute any of those assets or artwork.
&lt;/license team&gt;

**Comment by ZeroPointEnergy** [comment]
@ulm what about the License Agreement @Roesjka mentioned. It does look like that is the official licence agreement for game related resources in thirdparty tools like Pyfa. I'm not sure why the CCP dev did not mention that.

The licence agreement reads like it is directed at the developers of the tools. I'm not sure how that translates to the end-user of the application. Any ideas how to proceed?

**Comment by ulm** [comment]
Right, that is a license agreement specifically for developers. Note especially the fourth paragraph:
"YOU ARE RESPONSIBLE FOR ENSURING THAT (A) CCP HAS A VALID E-MAIL ADDRESS FOR YOU (AS DEVELOPER) AT ALL TIMES, ..."
This doesn't grant any rights to persons who haven't submitted their e-mail address to them (plus some additional conditions). Certainly it isn't GPL compatible, because already the right to distribute freely is missing.

**Comment by blitzmann** [comment]
@Roesjka @ZeroPointEnergy @ulm Indeed, the developer agreement is the agreement for using ESI (since it can write data to the EVE servers, this is much more restrictive than just a general "you can use the assets" agreement), at least how I understand it. The assets (which pyfa uses) is under a different license that doesn't require a formally accepted agreement.

> "CCP hf. has granted permission to pyfa to use EVE Online and all associated logos and designs for promotional and information purposes on its website"

>This appears to be a permission specific for their website. I don't think that it would allow us (nor anybody else) to distribute any of those assets or artwork.

It doesn't necessarily have to be a website... As stated above, that old license text was ripped from some old support article way back when, where CCP supplied the text with placeholders such as `CCP hf. has granted permission to <WEBSITE> to use EVE Online and all associated logos and designs for promotional and information purposes on its website`. Developers always just took that text and replaced the placeholders with their project name, so the fact that it says website may simply be a copy and paste issue.

As also stated, the newer "license" can be found here: https://community.eveonline.com/community/fansites/toolkit/

However it's it's in layman's speak. The only quotable thing there would be `©CCP hf. All rights reserved. Used with permission.`. Maybe I should just change to this? I dunno, IANAL, but please let me know if you guys need any assistance with this matter. it's never really been brought up before.

I've checked other programs that utilize the data (EVEMon, EVEHQ, EFT, Osmium), and the only one that really gives any sort of attribution is Osmium (but it's the same text as what pyfa has: https://o.smium.org/about)

**Comment by ZeroPointEnergy** [comment]
@blitzmann thank you for your investigations

@ulm so from what I read from that site here https://community.eveonline.com/community/fansites/toolkit/ it should be ok to just include the `©CCP hf. All rights reserved. Used with permission.`. Should we put that into a new license file and use that in the ebuild?

**Comment by titanofold** [comment]
There doesn't seem to be a license issue to be concerned about here that needs to be included by Gentoo or other distro because there isn't anything concerning end-users (other than telling the developer to not be evil...or else), but rather a copyright issue. The extent of which the developer (Pyfa) needs to post conspicuously. This is typically done in the About dialog, probably under a heading stating the copyright applies to the bundled images. (I can't check this, I'm not at home.)

From the [Developer License Agreement](https://developers.eveonline.com/resource/license-agreement), the copyright notice is to be stated as:

> © 2014 CCP hf. All rights reserved. "EVE", "EVE Online", "CCP", and all related logos and images are trademarks or registered trademarks of CCP hf.

What's really a concern here for the distros, is whether third-party redistribution is permitted. CCP doesn't explicitly forbid this, and seems to imply that it's okay. After all they do expect the developer to redistribute the materials along with the copyright notice. If they won't allow third-party redistribution, we (Gentoo) can solve this through a fetch restriction and I'm sure other distros have means of resolving this as well.
